### PR TITLE
Es: ....

### PR DESCRIPTION
**Pull request by Reixaldo**

### CODE REVIEW

**Comment by alama** [comment]
you did not update your copy. please merge upstream's ES branch

**Comment by alama** [comment]
SIGH, let me update your branch FOR you...

File: UI/quest_counter.csv [hunk 1]
```diff
@@ -335,13 +335,13 @@ TitleEnemy#0,"""Enemigos comunes"""
 TitleDropItem#0,"""Objetos disponibles"""
 TitlePartyLevel#0,"""Requisitos de misión"""
 TitleTimeLimit#0,"""Limite"""
-TitleEnemyLevel#0,"""Lvl Enemigos"""
+TitleEnemyLevel#0,"""Lvl enemigos"""
 QuestLevel#1,"""Nv. $(0)+"""
 QuestLevel_SubLimit#0,"""<pict mainclass_icon>Nv.$(0)～ <pict subclass_icon>Nv.$(1)～"""
 QuestLevelNon#0,"""No requiere Nv"""
-QuestKindSoro#0,"""En Solitario"""
+QuestKindSoro#0,"""En solitario""
```
Comment (alama) [inline UI/quest_counter.csv]: You removed a " at the end of this line

**Comment by Reixaldo** [comment]
thank you very much, for the corrections

**Pull request merged**